lib/cli/commands/serve.js

```diff
@@ -37,6 +37,11 @@ serve.builder = function(cli) {
 			default: "$HOME/.ui5/server/server.crt",
 			type: "string"
 		})
+		.option("sap-csp-policies", {
```

Comment:
The README.md needs to be updated with the new `--help` output for this flag as well.

Reply:
README.md still needs to be updated

Reply:
Done with 8d5f8fd2f2a95a56372831aed1112d012b9c4878